src/main/target/DIAT_MAMBAF405_2022B/target.h

```diff
@@ -0,0 +1,154 @@
+/*
+ * This file is part of EmuFlight. It is derived from Betaflight.
+ *
+ * This is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "DIAT"
+#define USBD_PRODUCT_STRING "MAMBAF405_2022B"
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_ACC_SPI_MPU6500
+#define USE_GYRO_SPI_MPU6500
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASHFS
+#define USE_FLASH_M25P16
+#define USE_MAX7456
+#define USE_BARO
+#define USE_ADC
+#define USE_SPI_GYRO
+// manual
+#define USE_VCP
+#define USE_SERIAL_4WAY_BLHELI_INTERFACE
+
+#define USE_LED
+#define LED0_PIN             PC15
+#define LED1_PIN             PC14
+#define BEEPER_PIN           PC13
+#define BEEPER_INVERTED
+
+#define USE_SPI
+#define USE_SPI_DEVICE_1
+#define SPI1_SCK_PIN         PA5
+#define SPI1_MOSI_PIN        PA6
+#define SPI1_MISO_PIN        PA7
+#define USE_SPI_DEVICE_2
+#define SPI2_SCK_PIN         PB13
+#define SPI2_MOSI_PIN        PB14
+#define SPI2_MISO_PIN        PB15
+#define USE_SPI_DEVICE_3
+#define SPI3_SCK_PIN         PC10
+#define SPI3_MOSI_PIN        PC11
+#define SPI3_MISO_PIN        PB5
+
+#define USE_EXTI
+
+#define GYRO_1_ALIGN CW270_DEG
+#define GYRO_1_CS_PIN        PA4
+#define GYRO_1_EXTI_PIN      PC4
+#define GYRO_1_SPI_INSTANCE SPI1
+
+#define USE_MPU_DATA_READY_SIGNAL
+
+#define ACC_MPU6000_ALIGN         GYRO_1_ALIGN
+#define GYRO_MPU6000_ALIGN        GYRO_1_ALIGN
+#define MPU6000_CS_PIN            GYRO_1_CS_PIN
+#define MPU6000_SPI_INSTANCE      GYRO_1_SPI_INSTANCE
+#define MPU_INT_EXTI              GYRO_1_EXTI_PIN
+
+#define ACC_MPU6500_ALIGN         GYRO_1_ALIGN
+#define GYRO_MPU6500_ALIGN        GYRO_1_ALIGN
+#define MPU6500_CS_PIN            GYRO_1_CS_PIN
+#define MPU6500_SPI_INSTANCE      GYRO_1_SPI_INSTANCE
+#define MPU_INT_EXTI              GYRO_1_EXTI_PIN
+
+#define ACC_ICM42688P_ALIGN       GYRO_1_ALIGN
+#define GYRO_ICM42688P_ALIGN      GYRO_1_ALIGN
+#define ICM42688P_CS_PIN          GYRO_1_CS_PIN
+#define ICM42688P_SPI_INSTANCE    GYRO_1_SPI_INSTANCE
+
```

Comment:
Since this is not defined as a dual gyro quad, and there is no `GYRO_2`, you cannot use defines for `GYRO_1`
```
#define GYRO_ICM42688P_ALIGN  CW270_DEG
#define ACC_ICM42688P_ALIGN  CW270_DEG
#define ICM42688P_CS_PIN       PA4
#define ICM42688P_EXTI_PIN        PC4
#define ICM42688P_SPI_INSTANCE SPI1

#define USE_MPU_DATA_READY_SIGNAL

#define ACC_MPU6000_ALIGN         ACC_ICM42688P_ALIGN
#define GYRO_MPU6000_ALIGN        GYRO_ICM42688P_ALIGN
#define MPU6000_CS_PIN            ICM42688P_CS_PIN
#define MPU6000_SPI_INSTANCE      ICM42688P_SPI_INSTANCE
#define MPU_INT_EXTI              ICM42688P_EXTI_PIN
```
Don't need MPU6500 defines, and probably don't need the MPU6000 defines since pretty certain this is an ICM42xxxp specific target and FC

Reply:
BF shows mpu6000, but not 6500

Reply:
nope, i'm wrong, i keep mixing up 722_2022B with 405_2022B :banana: 

BF:
```
#define BOARD_NAME        MAMBAF405_2022B
#define MANUFACTURER_ID   DIAT

#define USE_ACC
#define USE_ACC_SPI_MPU6000
#define USE_GYRO
#define USE_GYRO_SPI_MPU6000
#define USE_ACC_SPI_MPU6500
#define USE_GYRO_SPI_MPU6500
#define USE_GYRO_SPI_ICM42688P
#define USE_ACC_SPI_ICM42688P
```